sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/util/Utility.java

```diff
@@ -169,6 +173,14 @@ public static synchronized String formatCoordinate(double coordinate) {
         return COORDINATE_FORMATTER.format(coordinate);
     }
 
+    public static String readSynonymsFromFile(Path filePath) {
+        try {
+            return new String(Files.readAllBytes(filePath), StandardCharsets.UTF_8);
+        } catch (IOException ex) {
+            throw new ClientLogger(Utility.class).logExceptionAsError(new UncheckedIOException(ex));
```

Comment:
nit: is that better to have a static LOGGER? I know it is only used once but just want to discuss it.